2022/incoming/the-christmas-time-machine.pod

```diff
@@ -0,0 +1,81 @@
+Author: Thibault DUPONCHELLE <thibault.duponchelle@gmail.com>
+Title: The Christmas time machine
+Topic: Feature::Compat::Try
+
+=encoding utf8
+
+=head1 The Christmas time machine
+
+=head2 Terrible flashback at the line work
+
+At North Pole, elves are just starting the Christmas 2022 preparations.
+
+As usual they restarted doing line work, building toys, then sorting them per area. Everyone happily singing and working! When suddenly one elf put the first toy in a box marked "Salt Lake City" and got surprised it was not empty...
+
+And he got a terrible flashbak, they seemed to have forgotten to deliver 2021 toys in Salt Lake City!
```

Comment:
- flashback, missing `c`
- to deliver _the_ 2021 toys _to_ Salt Lake City!

Reply:
Edited